airflow/cli/commands/webserver_command.py

```diff
@@ -46,6 +46,8 @@
 
 class GunicornMonitor(LoggingMixin):
     """
+    GunicornMonitor
+
```

Comment:
```suggestion
```
The existing first paragraph _was_ a good summary.

Reply:
The existing first paragraph was good. But it's long as a first line summary .
Obeying D205 and E501 summary line needs to obey two rules.
- only 1 line (not 2+ line)
- length < 110